ouroboros-network/src/Ouroboros/Network/Diffusion/P2P.hs

```diff
@@ -1235,7 +1235,7 @@ nodeDataFlow :: NodeToNodeVersion
              -> NodeToNodeVersionData
              -> DataFlow
 nodeDataFlow v NodeToNodeVersionData { diffusionMode = InitiatorAndResponderDiffusionMode }
-                 | v >= NodeToNodeV_10
+                 | v >= NodeToNodeV_11
```

Comment:
TODO Since Conway's abrupt urgency was cancelled, I don't think we need to do this version bumping any longer. It should probably remain the case that v10 `Enable full duplex connections` (so this line of diff is now wrong) and thus we should add v11 to introduce Conway as we normally would have.

cc @coot agree?

Reply:
On Slack, @coot said:

> V10 is to be released as 1.35.5 which is still in progress.

I think that means this PR's diff should be simplified so that it leaves V10 alone and adds V11 for use with Conway.

cc @dnadales

Reply:
I reverted `V10` to the meaning it had before this PR and used `V11` for Conway.

Reply:
As a sanity check, I'm looking for occurrences of V10 in the diff between the base commit and the tip of your branch.

(`-U0` causes `git diff` to only show lines with changes, no context.)

```
$ git diff -U0 c607a8da7f origin/nfrisby/issue-3962-conway | grep -e V_11
+      , (NodeToNodeV_11, CardanoNodeToNodeVersion7) -- TODO: check this!
+- `NodeToNodeV_11` and `NodeToClientV_15`, both of which enable Conway.
+    | NodeToNodeV_11
+    encodeTerm NodeToNodeV_11 = CBOR.TInt 11
+    decodeTerm (CBOR.TInt 11) = Right NodeToNodeV_11
$ git diff -U0 c607a8da7f origin/nfrisby/issue-3962-conway | grep -e V_10
-      , (NodeToNodeV_10, CardanoNodeToNodeVersion6)
+      , (NodeToNodeV_10, CardanoNodeToNodeVersion7)
$
```

Is that second diff an oversight? Should that change be undone?

Reply:
You fixed this 👍; Resolving this thread.